src/Atoms/Popover/Popover.jsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import BasePopover from '@material-ui/core/Popover'
+import { css } from '@styled-system/css'
+import { layout } from 'styled-system'
+
+const Popover = styled(BasePopover)`
+  .MuiPopover-paper {
+    ${css({ border: '1px solid', borderColor: 'lightgrey', maxHeight: '440px' })};
```

Comment:
The css interpreter is overkill here, all the values are plain css, nothing from the theme or the props:
```suggestion
    border: '1px solid';
    borderColor: 'lightgrey';
    maxHeight: '440px';
```